tests/test_sso.py

```diff
@@ -60,6 +61,24 @@ def mock_connection(self):
             ],
         }
 
+    @pytest.fixture
+    def mock_connections(self):
+        return {
+            "data": [
+                {
```

Comment:
This looks like the schema for a Directory. Can we update the fields to be those for a Connection?